libs/ledger/src/upow/synergetic_execution_manager_interface.cpp

```diff
@@ -0,0 +1,56 @@
+//------------------------------------------------------------------------------
+//
+//   Copyright 2018-2019 Fetch.AI Limited
+//
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+//
+//------------------------------------------------------------------------------
+
+#include "ledger/upow/synergetic_execution_manager_interface.hpp"
+
+namespace fetch {
+namespace ledger {
+
+char const *ToString(SynergeticExecutionManagerInterface::ExecStatus status)
+{
+  using ExecStatus = SynergeticExecutionManagerInterface::ExecStatus;
```

Comment:
`using SynergeticExecutionManagerInterface::ExecStatus;`

Reply:
Done.

Reply:
Reverted - ExecStatus is an enum.

Reply:
Oh, sorry.